src/client/utils/getFormattedPrice.ts

```diff
@@ -0,0 +1,26 @@
+import { LocaleData } from 'src/client/l10n/locales'
+
+type FormatCurrencyParams = {
+  locale: LocaleData['currencyLocale']
+  amount: string
```

Comment:
hmm, why string? seems reasonable to expect it always as a number, no?

Reply:
Because we get a string from Giraffe, and I'd rather do the conversion to number here than everywhere we use this function.

Reply:
But maybe we should accept a `number` as well? I.e. `string | number`? 🤔

Reply:
Ahaaaa, just makes me a little worried if we let the graphql API direct how our implementation looks like

In my world, it should be the other way around :P (our code should make sense even without the API context)

But this is a bit nit-picky so perhaps a progressive approach is to allow both string + number input.